.circleci/config.yml

```diff
@@ -270,7 +270,7 @@ jobs:
           keys:
             - ship-integration-test-build-cache-base
       - run: |
-          export GO_DOWNLOAD_URL=https://dl.google.com/go/go1.13.linux-amd64.tar.gz
+          export GO_DOWNLOAD_URL=https://dl.google.com/go/go1.13.3.linux-amd64.tar.gz
```

Comment:
This'll fail unless `GO_SHA256SUM: 68a2297eb099d1a76097905a2ce334e3155004ec08cdea85f24527be3c48e856` above is updated

Reply:
I see. Thanks!